site/en/blog/progress-in-the-privacy-sandbox-2021-10/index.md

```diff
@@ -0,0 +1,263 @@
+---
+title: Progress in the Privacy Sandbox (October 2021)
+description: >
+  Monthly summary of Privacy Sandbox news including the renamed Federated
+  Credentials Management API and W3C TPAC sessions.
+layout: 'layouts/blog-post.njk'
+date: 2021-10-29
+authors:
+  - rowan_m
+hero: 'image/VWw0b3pM7jdugTkwI6Y81n6f5Yc2/TG00mBIgiEzZo09xjvBg.png'
+alt: >
+  Privacy Sandbox logo
+tags:
+  - progress-in-the-privacy-sandbox
+  - privacy
+---
+
+Welcome to the October edition of "**[Progress in the Privacy
+Sandbox](https://developer.chrome.com/tags/progress-in-the-privacy-sandbox/)**"
```

Comment:
Links within DCC should be absolute paths without the URL

Reply:
Oh, good spot. Pulled over from Markdown export.